### PR TITLE
When directory is specified, danger-swiftlint is still linting files outside the specified directory

### PR DESCRIPTION
**Pull request by victorg1991**
Right now, when I run danger-swiftlint with a custom directory and I have also modified files outside this directory, danger-swiftlint is linting all the files.

With this PR danger-swiftlint will only lint the modified files inside the specified directory :)

If this PR gets approved, can you upload a new version of the Gem? Thanks in advance!

Any comment or question would be appreciated

### CODE REVIEW

File: lib/danger_plugin.rb [hunk 1]
```diff
@@ -113,6 +115,8 @@ def find_swift_files(files=nil, excluded_paths=[])
         # Remove dups
         uniq.
         map { |file| File.expand_path(file) }.
+        # Ensure only files in the selected directory
+        select { |file| file.start_with?(dir_selected)}.
```
Comment (thii) [inline lib/danger_plugin.rb]: Shouldn't there be a space before `}`?

File: spec/danger_plugin_spec.rb [hunk 2]
```diff
@@ -68,18 +68,36 @@ module Danger
...
+        ])
+
+        expect_any_instance_of(Swiftlint).to receive(:lint)
+        .with(hash_including(:path => File.expand_path('spec/fixtures/some_dir/SwiftFile.swift')))
+        .once
+        .and_return(@swiftlint_response)
+
+        @swiftlint.lint_files
+          end
+
```
Comment (thii) [inline spec/danger_plugin_spec.rb]: Looks like indents are somewhat incorrect in this chunk.

**Comment by victorg1991** [comment]
I've addressed the comments @thii :)

Do you have permissions to upload the gem?

**Comment by thii** [comment]
@victorg1991 I don't :)
However, you can point your Gemfile to this repository in the mean time.

**Comment by victorg1991** [comment]
@thii even if I'm using this in danger?

**Comment by thii** [comment]
@victorg1991 Yes. Danger plugins are just normal Ruby gems.

**Comment by victorg1991** [comment]
@ashfurrow any chance to take a look into this? :)

Thanks!

**Comment by ashfurrow** [comment]
Ah, sorry for the delay. Will look into it now.

**Comment by ashfurrow** [comment]
Cool, this has been released as 0.9.0. Thanks again!

**Comment by victorg1991** [comment]
Wow so fast, thank you so much : D

**Comment by ashfurrow** [comment]
No worries! Feel free to ping me if I ever forget something 😅

**Pull request merged**